flight/util/json_parsing.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+
+def waypoint_parsing(filename: str):
+    f = open(filename, )
+    data_set = json.load(f)
+    # print(data_set)
+    f.close()
```

Comment:
You should also definitely use a context manager whenever possible
It can avoid headaches when it comes to resource management